Library/Formula/cpp-netlib.rb

```diff
@@ -0,0 +1,42 @@
+require "formula"
+
+class CppNetlib < Formula
+  homepage "http://cpp-netlib.org"
+  url "http://storage.googleapis.com/cpp-netlib-downloads/0.11.1/cpp-netlib-0.11.1RC2.tar.gz"
```

Comment:
My suggestion to switch to `https` from `http`, `storage.googleapis.com` supports it.